src/index.ts

```diff
@@ -271,5 +270,15 @@ function getRoutesFromControllers(
   }, []);
 }
 
+export const TestingFunctions =
+  process.env.NODE_ENV === "test"
+    ? {
+        getRoutesFromControllers,
+        getRoutesFromGroups,
+        returnContextResponse,
+        NotFound,
+      }
+    : null;
+
```

Comment:
instead of doing this you can move these functions to under `utils.ts` that would be better because `index.ts` is expose when package publish in npm but `utils.ts` not